README.md

```diff
@@ -339,3 +337,8 @@ To debug UAA and LDAP integrations, we use an OpenLdap docker image from [VMWare
 
 Use below command to clean-up container and volume:
 - `docker-compose down --volumes`
+
+# UAA's functionality as a SAML identity provider (IdP)
```

Comment:
I don't think we need to document in the README a feature that is no longer present. That's what the release notes are for. Removing the deprecation notice is a sufficient change.

Reply:
Removed the added section.